irida_import/irida_import.py

```diff
@@ -181,8 +181,13 @@ def get_sample_file(self, file_dict):
         resource = file_dict
         name = resource['fileName']
         path = resource['file']
+        links = file_dict['links']
+        for link in links:
```

Comment:
I am guessing your looping through as links is an array of objects? If so can we exit the loop early when we find the `rel = self` one since there should only be one instance. Can you actually change the if statement below to `if link['rel'] == 'self'`.

Reply:
Updated in f284834c611b2fa565ee03105e22c4fd87ffb518